scripts/DraconicEvolution.zs

```diff
@@ -11,7 +11,7 @@ import mods.jei.JEI.removeAndHide as rh;
 	recipes.addShapedMirrored("Mob Grinder", 
 	<draconicevolution:grinder>, 
 	[[<ore:ingotElectricalSteel>, <draconicevolution:draconium_block:1>, <ore:ingotElectricalSteel>],
-	[<astralsorcery:itemcrystalsword>.anyDamage(), <draconicevolution:draconic_core>, <astralsorcery:itemcrystalsword>.anyDamage()], 
+	[<astralsorcery:itemcrystalsword>.onlyDamageAtLeast(0), <draconicevolution:draconic_core>, <astralsorcery:itemcrystalsword>.onlyDamageAtLeast(0)], 

```

Comment:
Does this work? :D

Reply:
It's what is used for Rune of Sacrifice, and assuming the reason that one didn't break. I did test it out with freshly made swords and had no issue :D